sound/soc/sof/core.c

```diff
@@ -288,13 +308,21 @@ static int sof_probe(struct platform_device *pdev)
 		goto dbg_err;
 	}
 
+	/* optionally register virtio miscdev */
+	sof_virtio_miscdev_register(sdev);
+
 	/* init the IPC */
 	sdev->ipc = snd_sof_ipc_init(sdev);
 	if (!sdev->ipc) {
 		dev_err(sdev->dev, "error: failed to init DSP IPC %d\n", ret);
 		goto ipc_err;
 	}
 
+	sof_virtio_vfe_init(sdev, plat_data);
+	/* vFE will not touch HW. Let's skip fw loading */
```

Comment:
A new line should always precede a comment to make it obvious where the comment applies and to break the code into easier to follow blocks. This need fixed in several places.